k8s/crd-samples/deployment-cstor-pool.yaml

```diff
@@ -42,7 +46,15 @@ spec:
           mountPath: /dev
         - name: tmp
           mountPath: /tmp
-          mountPropagation: Bidirectional
+#          mountPropagation: Bidirectional
+        env:
+        - name: cstorid
+          value: "9368bb1c-5cfc-11e8-b6fb-54e1ad4a9dd4"
+
+        lifecycle:
+          postStart:
+            exec:
+              command: ["/bin/sh", "-c", "sleep 15"]
```

Comment:
It will really be great if we can provide comments on why a postStart lifecycle hook, etc? The yaml is an intent that expresses a lot of things. Yaml can accept comments. Lets provide these comments where required. We get a lot of questions on slack if we avoid these comments.

If there is a living design doc, lets put the link. However, this yaml should have comments explaining the rationale.

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova  @AmitKumarDas